gui/default/syncthing/folder/editFolderModalView.html

```diff
@@ -145,6 +146,9 @@ <h4 class="modal-title">
                     <span translate ng-if="folderEditor.trashcanClean.$error.min && folderEditor.trashcanClean.$dirty">A negative number of days doesn't make sense.</span>
                   </p>
                 </div>
+                <div class="form-group" ng-if="currentFolder.fileVersioningSelector=='systemtrash'">
+                  <p translate class="help-block">Files are moved to the system trash can when replaced or deleted by Syncthing.</p>
```

Comment:
This is DoS, anyone syncing a logfile, vm image, anything large that changes often will run out of space.